packages/client/playwright.config.ts

```diff
@@ -6,6 +6,10 @@ const config: PlaywrightTestConfig = {
     use: {
         headless: false,
         baseURL: 'http://localhost:4000',
+        viewport: {
```

Comment:
Reminder: ask why

Reply:
I had to modify the viewport so that the suggestion button does not hide first track cards.
Thanks to this modification, the last steps of test f can work: I'm targeting the score of the first track card, which was hidden by the suggestion button.

Reply:
Ok sounds like we should also take a decision about this button and its position in the screen